NOTICES.txt

```diff
@@ -1,586 +1,724 @@
-
-
---------------- APPENDIX 3: MIT License (Template) -----------
-
-Copyright <YEAR> <COPYRIGHT HOLDER>
-
-Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
-
-The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+=============== TABLE OF CONTENTS =============================
+
+The following is a listing of the Secrets Provider for K8s open source components detailed
+in this document. This list is provided for your convenience; please read
+further if you wish to review the copyright notice(s) and the full text
+of the license associated with each component.
+
+
+Section 1: Apache-2.0
+>>> github.com/cyberark/conjur-api-go - v0.6.0
+>>> github.com/cyberark/conjur-authn-k8s-client - v0.16.1
+>>> github.com/googleapis/gnostic - v0.3.1
+>>> github.com/modern-go/reflect2 - v1.0.1
+>>> k8s.io/api - v0.0.0-20190313235455-40a48860b5ab
+>>> k8s.io/apimachinery - v0.0.0-20190313205120-d7deff9243b1
+>>> k8s.io/client-go - v11.0.0
+>>> k8s.io/klog - v1.0.0
+>>> gopkg.in/yaml.v2 - v2.2.4
+>>> k8s.io/utils - v0.0.0-20191218082557-f07c713de883
+
+Section 2: MIT
+>>> github.com/cenkalti/backoff - v2.2.1
+>>> github.com/json-iterator/go - v1.1.9
+>>> github.com/smartystreets/goconvey - v0.0.0-20190731233626-505e41936337
+>>> sigs.k8s.io/yaml - v1.1.0
+
+Section 3: BSD 3-Clause
+>>> github.com/gogo/protobuf  - v1.3.1
+>>> golang.org/x/oauth2 - v0.0.0-20191202225959-858c2ad4c8b6
+>>> golang.org/x/time - v0.0.0-20191024005414-555d28b269f0
+>>> gopkg.in/inf.v0 - v0.9.1
+>>> sigs.k8s.io/yaml - v1.1.0
+
+
+APPENDIX: Standard License Files and Templates
+
+>>> Apache-2.0
```

Comment:
Again, why here was the order changed?

Reply:
I'm not sure the license order is as important, but I am updating to alphabetize those too.